### DISCUSSION TITLE
View/Display of failure.

### DISCUSSION THREAD

**Discussion by mrc4tt** [sarisia/actions-status-discord#208] [Ideas, Feature requests]
Hi.

Are it's possible for the status-discord can display/type what the failure are?

As example right now:
![](https://pic.lindhost.eu/LfsqBMuI6I.png)

After - where I want it to how, what happened, and what error on workflows (e.g)
![](https://pic.lindhost.eu/Vg4lXgHTrN.png)

Example failure:
![](https://pic.lindhost.eu/fOgvk2OYkz.png)

**Comment by sarisia** [comment]
Unfortunately, we do not have this feature at the moment.
GitHub does not provide execution logs to the action's execution environment.
Maybe we can get logs via GitHub API and make failure notification fancier, so I'll try to look for it.
